java/src/test/android/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+	ext.kotlin_version = "1.4.32"
+
+	repositories {
+		google()
+		jcenter()
+	}
+	dependencies {
+		classpath "com.android.tools.build:gradle:4.1.3"
+		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
You can choose to use 4.0.0 for com.android.tools.build:gradle such that it does not require gradle 6.5, such that you can revert the change to tools/ci_build/github/linux/ort_minimal/build_minimal_ort_and_run_tests.sh

Reply:
Also, if I change the gradle down to 4.0.0, then it seems like the NDK version will have a dismatch. It seems like 4.0.0 requires NDK version 21.0.6113669. Error message: `No version of NDK matched the requested version 21.0.6113669. Versions available locally: 21.4.7075529, 22.1.7171670` Any other idea how to fix this?

Reply:
Unfortunately this is an Android Gradle plugin bug started from 3.6 and was fixed in 4.1 https://issuetracker.google.com/issues/144111441

You may either downgrade the com.android.tools.build:gradle to 3.5.3, or revert to previous behavior